server/webhooks/imports/register-user.js

```diff
@@ -20,10 +21,27 @@ export function registerUser(user, transaction) {
   const userId = Accounts.createUser(userOptions);
   Accounts.addEmail(userId, email, true);
   Accounts.sendEnrollmentEmail(userId);
+  computeDistance(userId, userOptions);
 
   return userId;
 }
 
+function computeDistance(userId, { address, city, state, zip }) {
+  const origins = ['48.7335, -122.4873'];
+  const destinations = [`${address} ${zip}`];
+
+  gmaps.matrix(origins, destinations, function (err, distances) {
+    const distance = distances.rows[0].elements[0].distance.value
+    Meteor.users.update({ _id: user._id }, { $set: { distanceTraveled: distance } });
+  })
+}
+
+Meteor.methods({
```

Comment:
Remove this test method.  Ideally you would have added Unit tests for this functionality, but we dont have testing setup in this project yet... which is the worst.